request.coffee

```diff
@@ -2,17 +2,55 @@ _ = require 'lodash'
 request = require 'request'
 Promise = require 'bluebird'
 PinejsClientCore = require './core'
+BluebirdLRU = require 'bluebird-lru-cache'
 
 request = Promise.promisify(request)
 
+validParams = ['cache']
 module.exports = class PinejsClientRequest extends PinejsClientCore(_, Promise)
+	constructor: (params, backendParams) ->
```

Comment:
Shouldn't we also extend the angular sig to include `backendParams` to demonstrate that this is the expected sig there too? Otherwise now we have core with an unused var, this with both params which are used, and angular with just params (unless that's different somewhow)?